docs/messaging/gcp-pubsub.md

```diff
@@ -51,10 +68,70 @@ flowchart LR;
 | SpanKind | `PRODUCER` | `PRODUCER` | `CLIENT` |
 | Status | `Ok` | `Ok` | `Ok` |
 | `messaging.batch.message_count` |  |  | 2 |
-| `messaging.destination.name` | `"T"` | `"T"` | `"T"` |
+| `messaging.destination.template` | `"projects/P/topics/T"` | `"projects/P/topics/T"` | `"projects/P/topics/T"` |
 | `messaging.operation` | `"create"` | `"create"` | `"publish"` |
 | `messaging.message.id` | `"a1"` | `"a2"` | |
 | `messaging.message.envelope.size` | `1` | `1` | |
 | `messaging.system` | `"gcp_pubsub"` | `"gcp_pubsub"` | `"gcp_pubsub"` |
 
+### Unary Pull Example
+
+```mermaid
+flowchart TD;
+  subgraph CONSUMER
+  direction LR
+  R1[Receive m1]
+  SM1[Settle m1]
+  EM1[Extend m1]
+  end
+  subgraph PRODUCER
+  direction LR
+  CM1[Create m1]
+  PM1[Publish]
+  end
+  %% Link 0
+  CM1-. link .-PM1;
+  %% Link 1
+  CM1-. link .-R1;
+  %% Link 2
+  R1-. link .-SM1;
+  %% Link 3
+  R1-. link .-EM1;
+
+  %% Style the node and corresponding link
+  %% Producer links and nodes
+  classDef producer fill:green
+  class PM1,CM1 producer
+  linkStyle 0 color:green,stroke:green
+
+  %% Consumer links and nodes
+  classDef consumer fill:#032a61
+  class R1 consumer
+  linkStyle 1 color:#032a61,stroke:#032a61
```

Comment:
This shows a dark blue on a dark text, and it's really hard to read. Could we change to some other lighter bg color? ![image](https://github.com/open-telemetry/semantic-conventions/assets/5938087/30bceb1c-58e4-4f6f-ad47-7fe1b0d30221)

Reply:
Updated. Re-open if it still is hard to read.

Reply:
looks much better now, thank you!